client/my-sites/plan-features/list.jsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+class PlanFeaturesItemList extends Component {
+
+	render() {
+		return (
+			<ul className="plan-features__item-list">
+				{ this.props.children }
+			</ul>
+		);
+	}
+}
+
+export default PlanFeaturesItemList;
```

Comment:
Could we make this component (and the one above) stateless? :) Something like this:

```js
const PlanFeaturesItemList = ( { children } ) => {
	return (
		<ul className="plan-features__item-list">
			{ children }
		</ul>
	);
};

export default PlanFeaturesItemList;
```